transform/base_cnpj.go

```diff
@@ -0,0 +1,108 @@
+package transform
+
+import (
+	"fmt"
+	"io"
+	"path/filepath"
+)
+
+func addBaseCPNJ(dir string, l *lookups) error {
+	s, err := newSource(base_cpnj, dir)
+	if err != nil {
+		return fmt.Errorf("error creating source for partners: %w", err)
+	}
+	defer s.close()
+	for _, r := range s.readers {
+		for {
+			t, err := r.read()
+			if err == io.EOF {
+				break
+			}
+			if err != nil {
+				break // do not proceed in case of errors.
+			}
+			ls, err := filepath.Glob(filepath.Join(dir, t[0], "*.json"))
+			if err != nil {
+				return fmt.Errorf("error in the glob pattern: %w", err)
+			}
+			/*
+				if len(ls) == 0 {
+					return fmt.Errorf("No JSON file found for CNPJ base %s", t[0])
+				}
+			*/
+			for _, f := range ls {
+				c, err := companyFromJSON(f)
+				if err != nil {
+					return fmt.Errorf("error reading company from %s: %w", f, err)
+				}
+				err = c.fillMain(t, l)
+				if err != nil {
+					return fmt.Errorf("error filling company from %s: %w", f, err)
+				}
+				f, err = c.toJSON(dir)
+				if err != nil {
+					return err
+				}
+			}
+		}
+	}
+	return nil
+}
+
+func (c *company) fillMain(data []string, l *lookups) error {
```

Comment:
Em todo o projeto acho que não usamos _fill_, usamos? Podemos ou usar `add`, ou, como no código que usa tabelas de _look up_, só o nome mesmo. Também, ao invés de `data`, temos usado `r` (de _row_), e normalmente o `lookups` é o primeiro argumento…

```suggestion
func (c *company) mainCNPJ(l *lookus, r []string) error {
```